src/main/java/phrasecount/PhraseExporter.java

```diff
@@ -0,0 +1,25 @@
+package phrasecount;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.apache.accumulo.core.data.Mutation;
+import org.apache.fluo.recipes.accumulo.export.AccumuloExporter;
+import org.apache.fluo.recipes.core.export.SequencedExport;
+import phrasecount.pojos.Counts;
+import phrasecount.query.PhraseCountTable;
+
+/**
+ * Export code that converts {@link Counts} objects from the export queue to Mutations that are
+ * written to Accumulo.
+ */
+public class PhraseExporter extends AccumuloExporter<String, Counts> {
```

Comment:
I think there was a `PhraseExporter` before the the `AccumuloExport` recipe was developed.